.gitsubprojects

```diff
@@ -1,6 +1,5 @@
 # -*- mode: cmake -*-
 git_subproject(Servus https://github.com/HBPVIS/Servus.git 7a60583)
-git_subproject(ZeroEQ https://github.com/HBPVIS/ZeroEQ.git e6f91b0)
```

Comment:
Well, just disabling ZeroEQ in docker_build_wheel.sh would have been enough I suppose. Your fix also works of course.